freqtrade/freqtradebot.py

```diff
@@ -685,7 +685,7 @@ def check_and_call_adjust_trade_position(self, trade: Trade):
                 return
 
             remaining = (trade.amount - amount) * current_exit_rate
-            if min_exit_stake and remaining < min_exit_stake:
+            if min_exit_stake and (0 < remaining < min_exit_stake):
```

Comment:
Looking at this check due to the backtesting failure, i don't think it's correct.
we want to skip if
- remaining is negative (user provided a "too high" stake)
- remaining is between 0 and min_exit_stake.

We want it to pass however if remaining is 0 exactly.

as such - we need to either explicitly check for "remaining = 0" - or exclude 0 from the check somehow.

Reply:
@stash86 would be great if you could have a look at both the above comment, as well as on the actual change i did.

Reply:
sure